src/wp-admin/includes/update-core.php

```diff
@@ -770,6 +770,9 @@
 	'wp-includes/blocks/block/editor.min.css',
 	'wp-includes/blocks/block/editor-rtl.css',
 	'wp-includes/blocks/block/editor-rtl.min.css',
+	// 6.7
+	'wp-admin/images/browser-rtl.png',
+	'wp-admin/images/browser.png',
```

Comment:
The list of old files is updated at the end of the release cycle to ensure sites running trunk don't lose files in commits that are subsequently reverted (an edge case, I know).

```suggestion
```